docs/join-nodes.md

```diff
@@ -1,97 +1,90 @@
-# Join worker nodes
+# Join a worker node
 
-Joining worker nodes is pretty much the exact same process as with [k0s](https://docs.k0sproject.io/stable/k0s-multi-node/#4-add-workers-to-the-cluster) in general. You need a join token that enables mutual trust between the worker and controller(s) and which allows the node to join the cluster as worker.
+Joining worker nodes to your control plane is similar to the process of
+[joining worker nodes with k0s](https://docs.k0sproject.io/stable/k0s-multi-node/#4-add-workers-to-the-cluster)
+in general. You need to a join token that establishes mutual trust between
```

Comment:
I would remove "in general" as it serves no purpose. 
You need to a join token -> You need to join token